crates/pessimistic-proof-core/src/proof.rs

```diff
@@ -6,23 +6,29 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 #[cfg(not(target_os = "zkvm"))]
 use tracing::warn;
+#[cfg(target_os = "zkvm")]
+use unified_bridge::aggchain_proof::AggchainProofPublicValues;
 use unified_bridge::global_index::GlobalIndex;
-use unified_bridge::imported_bridge_exit::{commit_imported_bridge_exits, Error};
+use unified_bridge::imported_bridge_exit::Error;
+use unified_bridge::imported_bridge_exit::ImportedBridgeExitCommitmentValues;
+use unified_bridge::CommitmentVersion;
 use unified_bridge::{
     bridge_exit::NetworkId, local_exit_tree::LocalExitTreeError, token_info::TokenInfo,
 };
 
-#[cfg(target_os = "zkvm")]
-use crate::aggchain_proof::AggchainProofPublicValues;
 use crate::{
     aggchain_proof::AggchainData,
     local_state::{
-        commitment::PPRootVersion, commitment::PessimisticRoot, commitment::SignatureCommitment,
+        commitment::PessimisticRoot, commitment::SignatureCommitmentValues,
         commitment::StateCommitment, NetworkState,
     },
     multi_batch_header::MultiBatchHeader,
 };
 
+/// Version of the commitment on the imported bridge exits to verify the
+/// aggchain proof.
+pub const IMPORTED_BRIDGE_EXIT_COMMITMENT_VERSION: CommitmentVersion = CommitmentVersion::V2;
```

Comment:
should be v3 right?

Reply:
yeah as discussed f2f taking it back to V3 as I set it in the PR in the `provers` side.

rational: the commitment in the certificate signature and the one in the aggchain proof public values does not need to be in the same version, so keeping it v2 or v3 changes nothing in terms of easing the integration process